### PR TITLE
Landing page regression tests

### LINKED ISSUE #410: Integrate playwright tests for landing page
We should have following tests:

- [x] Page should be viewed without any error
- [x] All links should work (social media links and others too)
- [x] Navbar should be visible and all redirection should work

### PR DESCRIPTION
**Pull request by Eren-Yeaager**
Resolves 
#410  and #384

### CODE REVIEW

File: playwright-tests/tests/landing-page.spec.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    await page.goto(`/${ROOT_SRC}`);
+  });
+  test("User is not logged in", async ({ page }) => {
+    const startProjectButton = page.getByRole("button", {
+      name: "Start Project",
+    });
+    await expect(startProjectButton).toBeVisible();
+    await startProjectButton.click();
+    expect(page.url()).toContain("?page=projects&tab=editor");
+  });
```
Comment (Megha-Dev-19) [inline playwright-tests/tests/landing-page.spec.js:65]: Elliot has changed this to show login screen on editor when user is not signed in, let's update the code to check that login action, once his PR is merged
and let's another test when user is logged in and they click on "Start Project".

File: playwright-tests/tests/landing-page.spec.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+      name: "Explore Project",
+    });
+    await expect(exploreProjectsButton).toBeVisible();
+    await exploreProjectsButton.click();
+    expect(page.url()).toContain("?page=projects");
+  });
+});
+
+test.describe("Explore Projects for logged out user", () => {
+  test.use({
```
Comment (Megha-Dev-19) [inline playwright-tests/tests/landing-page.spec.js:86]: let's have two tests with subtests like this: 
1) User not logged in:
     - Start Project action
2) User logged in: 
     - Start Project action
     - Explore project action
     - all other links redirection
     - Social Media links
(you can checkout Elliot's PR for reference)

**Review by Megha-Dev-19 — COMMENTED** [review]
@Eren-Yeaager great job with the tests.
Let few comments.

File: playwright-tests/tests/landing-page.spec.js (hunk 1 above)
Comment (Eren-Yeaager) [reply playwright-tests/tests/landing-page.spec.js:65]: Sure will add that assertion to the test after the PR is merged

File: playwright-tests/tests/landing-page.spec.js (hunk 2 above)
Comment (Eren-Yeaager) [reply playwright-tests/tests/landing-page.spec.js:86]: Sure will implement this structure.

File: playwright-tests/tests/landing-page.spec.js (hunk 1 above)
Comment (elliotBraem) [reply playwright-tests/tests/landing-page.spec.js:65]: Merged @Eren-Yeaager

File: playwright-tests/tests/landing-page.spec.js (hunk 2 above)
Comment (Eren-Yeaager) [reply playwright-tests/tests/landing-page.spec.js]: Implemented it.

File: playwright-tests/tests/landing-page.spec.js (hunk 1 above)
Comment (Eren-Yeaager) [reply playwright-tests/tests/landing-page.spec.js]: Added the assertion

**Pull request merged**